sdk/storage/Azure.Storage.Common/swagger/Generator/src/models.ts

```diff
@@ -151,7 +151,8 @@ export interface IProperty {
     required?: boolean,
     readonly: boolean,
     xml?: IXmlSettings,
-    model: IModelType
+    model: IModelType,
+    isNullable?: boolean
```

Comment:
Does this need to be `isNullable?` or can it just be `isNullable`?

Reply:
I really wanted to not have this as an optional parameter but since TypeScript doesn't assign default values for primitive types, I had to make it as optional (https://github.com/Microsoft/TypeScript/issues/5113)

If I'd make `isNullable` required then we would have to set `isNullable` when we add [headers ](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/swagger/Generator/src/serviceModel.ts#L955) and [body ](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/swagger/Generator/src/serviceModel.ts#L967) in [createResponseType()](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/swagger/Generator/src/serviceModel.ts#L913) method.

We can set `false` in those cases but do we need to set `isNullable` when creating a response type?